src/app/prototype/tables/page.tsx

```diff
@@ -0,0 +1,278 @@
+import { Button } from '@/components/uswds/Button';
+import Link from 'next/link';
+import classnames from 'classnames';
+import { Table } from '@/components/uswds/Table/Table';
+import { TableHead } from '@/components/uswds/Table/TableHead';
+import { TableHeadCell } from '@/components/uswds/Table/TableHeadCell';
+import { TableBody } from '@/components/uswds/Table/TableBody';
+import { TableRow } from '@/components/uswds/Table/TableRow';
+import { TableCell } from '@/components/uswds/Table/TableCell';
+
+export default function TablePage() {
+  // active color: bg-accent-cool-lightest
+
+  const mockUsers = [
+    {
+      email: 'firstname.lastname@gsa.gov',
+      orgRoles: '4 roles',
+      spaceRoles: '2 of 3 spaces',
+      expires: 'Never',
+      lastLogin: 'Oct. 24, 2023',
+    },
+    {
+      email: 'longerfirstname.lastname@gsa.gov',
+      orgRoles: 'None yet — edit roles',
+      spaceRoles: 'None yet — edit permissions',
+      expires: '90 days',
+      lastLogin: 'Unknown — resend invite',
+    },
+  ];
+
+  const SortArrow = ({
+    direction = 'unsorted',
+  }: {
+    direction?: 'unsorted' | 'asc' | 'desc';
+  }) => (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      width="24"
+      height="24"
+    >
```

Comment:
[non-blocking comment for this PR]

Small accessibility suggestion: these svgs might benefit from being wrapped in `button` elements, with some accessible labels that describe the current direction of the column sort.

(It’s possible the USWDS sorting library might take care of this for us. In which case, please ignore this comment!)

Reply:
They're buttons now for the Table version—I haven't updated the Layout Grid version with the new SortButton component. Try out the table column labels in a screenreader and LMK what you think. I tried to make them as close to USWDS ones as possible. Something I found in both the USWDS table and ours is that sorting states might be repeated when reading data cells. Since it happens the same way on USWDS table, I chalked it up to the screen reader.

Reply:
> They're buttons now for the Table version

Oops, sorry Eleni — I was looking at the component, not the rendered source. Mea culpa.

> Try out the table column labels in a screenreader and LMK what you think. 

@echappen and I chatted about this, and agreed the table sounds a little too verbose when navigating through each cell. (UX-wise, I wouldn’t expect the sort status of the column to be read aloud each time.) But since our table matches the output in their sortable table component, we’re going to check with the USWDS team.